.github/workflows/latest-release.yml

```diff
@@ -75,8 +75,8 @@ jobs:
         run: |
             value=`cat STABLE-RELEASE`
             if [ ${{ github.ref }} == refs/heads/$value ]; then
-              docker pull docker.io/accuknox/knoxautopolicy:$value
-              docker image accuknox/knoxautopolicy:$value accuknox/knoxautopolicy:stable
+              docker pull docker.io/kubearmor/knoxautopolicy:$value
+              docker image kubearmor/knoxautopolicy:$value kubearmor/knoxautopolicy:stable
```

Comment:
lets keep the docker hub repo to accuknox. If we change this then we have to change the github repo secrets as well.